### PR TITLE
Make the display of PR number in tree view configurable

### LINKED ISSUE #4463: Missing pull request number on a head of title
Please tell me how to show number on a head of title if this issue is specification.

- Extension version: [0.56.0](https://github.com/microsoft/vscode-pull-request-github/releases/tag/v0.56.0)
- VSCode Version: Version: 1.74.3, Commit: 97dec172d3256f8ca4bfb2143f3f76b503ca0534
- OS: macOS Monterey 12.6.2 (21G320)

Steps to Reproduce:

1. Show pull requests in the side bar

### 0.56.0

Missing PR number

Screen Shot 2023-01-26 at 12 13 26

### 0.54.1

Shown PR number

Screen Shot 2023-01-26 at 12 12 26

### PR DESCRIPTION
**Pull request by kabel**
Closes #4463

~~Also updates the description node to always show the PR number, rather than just in the tooltip.~~

### CODE REVIEW

File: src/view/treeNodes/pullRequestNode.ts [hunk 1]
```diff
@@ -64,7 +65,7 @@ export class PRNode extends TreeNode implements vscode.CommentingRangeProvider {
 		try {
 			const descriptionNode = new DescriptionNode(
 				this,
-				vscode.l10n.t('Description'),
+				`${vscode.l10n.t('Description')} #${this.pullRequestModel.number}`,
```
Comment (alexr00) [inline src/view/treeNodes/pullRequestNode.ts]: I'm not a fan of having the PR number in the description node. When `githubPullRequests.showPullRequestNumberInTree` is false, it feels noisy and when `githubPullRequests.showPullRequestNumberInTree` is true it is redundant. I've pushed a change to revert this.

**Review by alexr00 — COMMENTED** [review]
Looks good, thank you for the new setting!

**Review by alexr00 — APPROVED** [review]
I will merge after the release tomorrow!

File: src/view/treeNodes/pullRequestNode.ts (hunk 1 above)
Comment (kabel) [reply src/view/treeNodes/pullRequestNode.ts]: Fair enough.

**Comment by prenaux** [comment]
Thanks a lot for adding this :)

**Pull request merged**